ingest/src/main/scala/hydra.ingest/app/AppConfig.scala

```diff
@@ -82,7 +88,10 @@ object AppConfig {
       env("HYDRA_KAFKA_PRODUCER_BOOTSTRAP_SERVERS").as[String],
       env("HYDRA_DEFAULT_PARTIONS").as[Int].default(10),
       env("HYDRA_REPLICATION_FACTOR").as[Short].default(3),
-      env("HYDRA_MIN_INSYNC_REPLICAS").as[Short].default(2)
+      env("HYDRA_MIN_INSYNC_REPLICAS").as[Short].default(2),
+      env("TIMESTAMP_VALIDATION_CUTOFF_DATE_IN_YYYYMMDD")
+        .as[Instant]
+        .default(dateStringToInstant("20230711"))
```

Comment:
Just use already date and convert to Instant type.

Reply:
Updated date to Instant type.